### ISSUE TITLE
[storage] : Change APIs according to `Resource-oriented design` https://google.aip.dev/121

### ISSUE THREAD

**Issue by glimchb** [opiproject/opi-api#174]
After #171 is done we need a separate PR to fix all storage according to https://google.aip.dev/121

- https://google.aip.dev/131 - Get
- https://google.aip.dev/132 - List
- https://google.aip.dev/133 - Create
- https://google.aip.dev/134 - Update
- https://google.aip.dev/135 - Delete

and https://google.aip.dev/127 - `HTTP and gRPC Transcoding`

**Comment by llabordehpe** [comment]
Do you want to follow Google's AIPs just for Create (your link https://google.aip.dev/133), or also for Update, Delete, Get, etc... ?

**Comment by glimchb** [comment]
yes, absolutely for everything @llabordehpe just gave `Create` as an example

**Comment by glimchb** [comment]
After #218:
```
+--------------------------------------+------------------+----------------+
|                 RULE                 | TOTAL VIOLATIONS | VIOLATED FILES |
+--------------------------------------+------------------+----------------+
| core::0156::forbidden-methods        |               11 |              7 |
| core::0127::http-annotation          |               12 |              7 |
| core::0127::resource-name-extraction |               22 |              7 |
| core::0192::has-comments             |              380 |              8 |
+--------------------------------------+------------------+----------------+
```
